src/main/java/com/shaft/gui/browser/BrowserActions.java

```diff
@@ -302,6 +303,43 @@ public static void switchToWindow(WebDriver driver, String nameOrHandle) {
         FluentBrowserActions.getInstance().switchToWindow(nameOrHandle);
     }
 
+    /**
+     * Switches focus to default content, is mainly used in coordination with
+     * {@link #switchToIframe(WebDriver, By)} to exit any iFrame layer and go back
+     * to the main page
+     *
+     * @param driver the current instance of Selenium WebDriver
+     */
+    @Deprecated
```

Comment:
Let's not add any deprecated methods please since we're only a few months away from deleting this class. 🙏